src/pages/react/Installations/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: React App Dependencies Installation 
+---
+## Pre-requisites 
+We need the right combinations of `Babel` ,` Webpack` and `NPM `dependencies to get started with a basic React Application.  `create-react-app` a command line tool is provided by React developers to abstract all the complexities involved in setting up a react project.
+What goes under the hood when we use this command is what we will be explaining here.
```

Comment:
This is a fairly bold statement to have, and by the end of the article I would not have a clue what was going on. All I would have is a bunch of dependencies in a `node_modules` folder.

You have not described how to hook up babel + webpack + react + react-dom + jest + eslint + all the other things that `create-react-app` does for us.